lib/heuristics/scheduling_heuristic.rb

```diff
@@ -1044,11 +1046,21 @@ def prepare_output_and_collect_routes(vrp)
       @candidate_routes.each{ |_vehicle, all_days_routes|
         all_days_routes.keys.sort.each{ |day|
           route = all_days_routes[day]
+
           computed_activities = []
+          start_time, end_time = if route[:current_route].empty?
+            computed_activities << get_stop(vrp, route[:start_point_id]) if route[:start_point_id]
+            computed_activities << get_stop(vrp, route[:end_point_id]) if route[:end_point_id]
 
-          computed_activities << get_stop(vrp, route[:start_point_id]) if route[:start_point_id]
-          computed_activities += get_activities(day, vrp, route[:current_route])
-          computed_activities << get_stop(vrp, route[:end_point_id]) if route[:end_point_id]
+            [0, 0]
```

Comment:
Are these 0's relative to the start time of the vehicle or absolute (i.e., they will be shown on the solution file)? For ortools `start_time` is equal to a "valid" time (earliest_start for example) and end_time too even if the route is empty.